### DISCUSSION TITLE
User documentation

### DISCUSSION THREAD

**Discussion by frankkloprogge** [OpenVoiceOS/ovos-buildroot#287] [General]
Hi, I would like to create my own skill in ovos but I can't find any documentation how I should do this. The only documentation I can find is the technical manual which is nice too but not of any use to me at the moment.
I'm using a rpi4. installed bookworm lite and then ran the ovos installer.

**Comment by JarbasAl** [comment]
the technical manual are the official docs and contain a lot of info about skill development

![image](https://github.com/user-attachments/assets/8b304495-b1e5-487f-9fe3-83a770e255b8)

if that doesnt help you im afraid there are no alternatives

**Reply by frankkloprogge** [reply]
Hi Jarbas,

I posed the wrong question. What I really would like to have is documentation about creating the framework for a skill. I have been using picroft for a while and now I'm switching to ovos so I know how to handle intents and dialogs. I just can't find a tool like "mycroft-msm" ( I believe it was called). Or should I just copy a skill and start editing it ? But in that case : do I need to copy more than just the directory with the files ?

**Reply by frankkloprogge** [reply]
Oh: I forgot to wish you a "Merry Christmas" !

**Comment by devvmh** [comment]
On a somewhat-related note, I see a lot of links around that point to https://openvoiceos.github.io/community-docs/; does this not exist anymore?
